packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -430,6 +442,11 @@ export interface IMenuItemStyles extends IButtonStyles {
   * Styles for a divider item of a ConextualMenu.
   */
   divider: IStyle;
+
+  /**
+   * Styles for a split button divider in a menu item
+   */
+  splitDivider: IStyle;
```

Comment:
I wonder if we want to clarify some terminology on horizontal dividers vs vertical dividers. Perhaps we should be referring to the vertical dividers as separators instead to avoid confusion. Thoughts @dzearing?

Reply:
The same comment applies to the PR I have open to introduce a generic vertical divider that is center aligned.

Reply:
I agree that this can be confusing, for now I renamed to splitVerticalDivider